web-client/src/pages/contact.jsx

```diff
@@ -1,33 +1,40 @@
 import React from 'react';
 import '../App.css';
-import Informations from '../components/information'
-import GoogleMaps from '../components/maps'
+import ContactInfo from '../components/ContactInfo'
+import ContactMap from '../components/ContactMap'
 import picture from '../images/empty-image.png'
 
 
 function Contact(props) {
   const inf1 = {
     path: picture,
+    street: false,
     link: "mailto:conference@gmail.com",
-    name: "conference@gmail.com" 
+    name: "conference@gmail.com",
+    icon: "Email icon"
   };
 
   const inf2 = {
     path: picture, 
+    street: false,
     link: "tel:987654321",
-    name: "987654321" 
+    name: "987654321",
+    icon: "Phone icon"
   };
 
   const inf3 = {
     path: picture,
-    link: "#",
-    name: "Parkowa 11/12" 
+    street: true,
```

Comment:
is that necessary?

Reply:
Yes, pls look file ContactInfo

Reply:
probably nobody does that but you can add link to google maps instead

Reply:
For me something is wrong with this solution. I don't see any sense in storing boolean value only in a situation where developer doesn't provide link property (what's happens when street: false and link is not set?).

You should only check status of link property (there're some tricks with '!')

Or

As @upsu90 said before => assign link of google maps (for this address)